### ISSUE TITLE
`source-repository` vs `source-repository-package`

### ISSUE THREAD

**Issue by philderbeast** [haskell/cabal#9655]
**What is wrong with the docs?**
I'm familiar with `source-repository-package` in projects. I'm reading about the `source-repository` stanza of a package. The section title "Source Repositories" is plural.

https://github.com/haskell/cabal/blob/ae3c40a20bf98870488e3b40fc4495009b026e33/doc/cabal-package-description-file.rst#L2668-L2676

I saw that and immediately wondered what use would there be in one package for multiple `source-repository` stanzas?

Can we be clearer that all `source-repository` elements are for the one package and not for dependencies?

**Comment by philderbeast** [comment]
I ran a quick `cabal check` and we can have as many `source-repository` stanza as we please. The following passes `cabal check`;

```diff
$ git diff
diff --git a/cabal-install/cabal-install.cabal b/cabal-install/cabal-install.cabal
index f842d4d31..afc1101b9 100644
--- a/cabal-install/cabal-install.cabal
+++ b/cabal-install/cabal-install.cabal
@@ -26,6 +26,23 @@ source-repository head
   location: https://github.com/haskell/cabal/
   subdir:   cabal-install
 
+source-repository head
+  type:     git
+  location: https://gitlab.com/haskell/cabal/
+  subdir:   cabal-install
+
+source-repository this
+  type:     git
+  location: https://github.com/haskell/cabal/
+  tag:      v3.11.0.0
+  subdir:   cabal-install
+
+source-repository this
+  type:     git
+  location: https://gitlab.com/haskell/cabal/
+  tag:      v3.11.0.0
+  subdir:   cabal-install
```

**Comment by philderbeast** [comment]
The docs have been fixed by #9701.
